docs/atlascli/command/atlas-clusters-search-indexes-describe.txt

```diff
@@ -93,8 +93,8 @@ If the command succeeds, the CLI returns output similar to the following sample.
 
 .. code-block::
 
-   ID          NAME     DATABASE     COLLECTION
-   <IndexID>   <Name>   <Database>   <CollectionName>
+   ID          NAME     DATABASE     COLLECTION         TYPE
+   <IndexID>   <Name>   <Database>   <CollectionName>   {{or .Type "search">
```

Comment:
FYI this seems odd @tibulca

Reply:
yes, noticed it too and fixed it in the last version